### DISCUSSION TITLE
kiali basic auth with jaeger

### DISCUSSION THREAD

**Discussion by rpuserh** [kiali/kiali#6018] [Q&A] [answered]
Hi,

I have Jaeger (tempo) backend which has basic auth. How do I configure Kiali to auth with Jaeger using basic auth?
I'm open to any other auth methods that might simplify auth process

Configuration [docs](https://kiali.io/docs/configuration/p8s-jaeger-grafana/jaeger/) reference has no option for auth

Please advice.

**Comment by jmazzitelli** [comment, marked as answer]
_moved to Q&A discussions_

For Jaeger auth config, see the Kiali CR reference doc: https://kiali.io/docs/configuration/kialis.kiali.io/#.spec.external_services.tracing

If you are looking for specific Tempo support, that is not supported yet. See this issue: https://github.com/kiali/kiali/issues/5850
